paymentsheet/api/paymentsheet.api

```diff
@@ -1,3 +1,7 @@
+public final class com/stripe/android/paymentsheet/AddPaymentMethodsAdapterKt {
```

Comment:
Can these compose functions be made internal?

Reply:
Yes. That was an oversight.